src/Esfa.Vacancy.Application/Commands/CreateApprenticeship/Validators/WageTypeValidator.cs

```diff
@@ -27,6 +39,14 @@ private void ConfigureWageTypeValidator()
                 RuleFor(request => request.WageTypeReason)
                     .Null()
                     .WithErrorCode(ErrorCodes.CreateApprenticeship.WageTypeReason);
+
+                RuleFor(request => request.WageUnit)
+                    .Cascade(CascadeMode.StopOnFirstFailure)
+                    .NotEqual(WageUnit.NotApplicable)
+                    .WithErrorCode(ErrorCodes.CreateApprenticeship.WageUnit)
+                    .MustAsync(BeGreaterThanOrEqualToApprenticeshipMinimumWage)
```

Comment:
Is this going to be refactored under amount field later?

Reply:
I'd have to move lines 43 - 46 with it wherever it goes as this rule only applies when WageUnit is not NotApplicable.